examples/examples_test.go

```diff
@@ -165,7 +165,7 @@ func TestExampleObjectSchemas(t *testing.T) {
 		"../cmd/integration": {
 			"v1-controller": &api.ReplicationController{},
 		},
-		"../examples/guestbook": {
+		"../examples/guestbook/test-data": {
```

Comment:
We want the examples test to test the configs actually used by the example. That's why the test exists. 

We need to extend it to handle multiple resources in one file.